windowsforms/MSIX-WindowsForms/README.md

```diff
@@ -0,0 +1,51 @@
+# MSIX Windows Forms Core Application
+
+This sample shows how to use the Windows Packaging Project to package a WinForms application running on Core.
+
+## Pre-requisites
+
+To use the Windows Packaging Project (wapproj) you need to install the Universal Windows workload in Visual Studio.
+
+To produce `MSIX` packages you must have the **Windows 10 October 2018 SDK** (aka 10.0.17763), otherwise the generated package will have the `APPX` extension.
+
+To install MSIX packages you need  the **Windows 10 October 2018 Update**
+
+## Customize the Packaging Project
+
+Currently, the packaging project does not support .NET Core applications, as a workaround we updated the project files to call the `publish` target to produce a self-contained app, and fix the manifest entry point. 
```

Comment:
- period after ".NET Core applications"
- as a workaround we --> As a workaround, we
- and fix --> and to fix